documentation/sonic/ZTP.md

```diff
@@ -51,7 +31,7 @@ option bootfile-name "http://<http-server>/provision.json";
 Refer to the [docs](https://github.com/sonic-net/SONiC/blob/master/doc/ztp/ztp.md) for detailed information on how to
 create ZTP boot file.
 
-See and example of `provision.json` file (for full list of supported ZTP plugins like snmp, firmware, etc. and available
+See and example of `ztp.json` file (for full list of supported ZTP plugins like snmp, firmware, etc. and available
```

Comment:
I think this should be "See an example"

Reply:
fixed 55160baac7e9aac42a6c6945c50be283992c1391